.github/workflows/e2e-tests.yml

```diff
@@ -125,9 +125,15 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       openldap:
-        image: osixia/openldap:1.5.0
+        image: bitnami/openldap:2.6.4
```

Comment:
There's a reference to the old image in a few more places in the code base.
Can you please update it everywhere?

Reply:
I've updated the command in the comment in `e2e-ldap-helpers.js`. I don't see references to it anywhere else but let me know if I'm missing something

Reply:
Sorry, didn't see your reply before this got auto-merged. Fixing that in https://github.com/metabase/metabase/pull/32645